app/graphql/types/lists_collection_type.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Types
+  class ListsCollectionType < Base::Object
```

Comment:
please remove this type, lists should be resolved using the connection type (for pagination)